.github/vendor/github.com/grafana/loki-release/workflows/build.libsonnet

```diff
@@ -105,6 +105,152 @@ local releaseLibStep = common.releaseLibStep;
       }),
     ]),
 
+  dockerPlugin: function(
```

Comment:
this is vendored from `grafana/loki-release`. we cannot make the change here or it will get wiped out.